pkg/subnet/local.go

```diff
@@ -246,7 +244,19 @@ func (d *LocalSubnetDeployer) doDeploy(chain string, chainGenesis string) error
 	ux.Logger.PrintToUser("Network name:     %s", chain)
 	ux.Logger.PrintToUser("Chain ID:         %s", chainID)
 	ux.Logger.PrintToUser("Currency Symbol:  %s", tokenName)
-	return nil
+
+	// we can safely ignore errors here as the subnets have already been generated
+	subnetID, _ := ids.FromString(subnetIDStr)
+	var blockchainID ids.ID
+	// TODO: the specs say CustomVms is a map by VM ID,
```

Comment:
Can we get a ticket for this

Reply:
not a bug. the comment is obsolete. will fix in subsecuent ANR PR. to be included into ANR refactoring PRs.

Reply:
https://ava-labs.atlassian.net/browse/AV-1918

Reply:
OK. Removed the TODO comment